docs/versioned_docs/version-zenith/.gitattributes

```diff
@@ -0,0 +1 @@
+reference/** linguist-generated
```

Comment:
I placed it here instead of closer to the root with a longer path because I think it'll make it easier to re-structure the docs files in the future.